scripts/go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/pulumi/pulumi-gcp/scripts
 
 go 1.13
 
-require github.com/pulumi/pulumi/pkg v0.0.0-20200323215703-1d4a75674b2e
+require github.com/pulumi/pulumi/pkg v0.0.0-20200322194843-61928f04e052
```

Comment:
This seems to be going back? Is that intended? Why so many changes in this PR?

Reply:
The original PR had picked up an incorrect version of the upstream API causing it to throw a build failure

The issue was resolved by using a replace in the go.mod

This version of pulumi/pkg makes everything the same version in the report now